src/main/resources/default/assets/scripts/blob-upload-url-popover.js

```diff
@@ -0,0 +1,106 @@
+/**
+ * Inits the url upload popover.
+ *
+ * Uses jQuery to init bootstraps popover.
+ *
+ * @param triggerElement The element (e.g. Button) that triggers the popover to enter an URL
+ * @param uploadContainer The container the triggerElement and other components belongs to
+ * @param resetButton The button that resets the upload field
+ * @param i18n Contains information about the label, the ok-Button and the cancel-Button of the popover
+ */
+function initUrlUploadPopover(triggerElement, uploadContainer, resetButton, i18n) {
+    $(triggerElement).popover({
```

Comment:
I thought we want to get away from jquery

Reply:
This is also my current state of knowledge. Therefore I have tried as far as possible not to use jQuery. Unfortunately we need the popover functionality of Bootstrap here. Is there a vanilla-js way to call a bootstrap popover?